qcodes/dataset/sqlite/database.py

```diff
@@ -190,7 +190,7 @@ def get_db_version_and_newest_available_version(path_to_db: str) -> Tuple[int,
 
 
 def get_DB_location() -> str:
-    return expanduser(qcodes.config["core"]["db_location"])
+    return normpath(expanduser(qcodes.config["core"]["db_location"]))
```

Comment:
is `get_DB_location` used everywhere in qcodes? in other words, are there other places where "normpath(expanduser" would be added?

Reply:
I think so. The exanduser thing is a bit of a pain point. So we have put it here